Remove foundation-sites npm package (#16889)
decidim/decidim: RELEASE_NOTES.md

## 5. Changes in APIs

### 5.1. `initFoundation` Javascript function has been removed

> In order to fully remove foundation, we need to remove any dependency to Foundation-Sites. In the latest releases we started to rely more on Stimulus controllers and plain Javascript.

Reviewer: ```suggestion
In order to fully remove Foundation CSS, we need to remove any dependency to Foundation-Sites. In the latest releases we started to rely more on Stimulus controllers and plain Javascript.
```

> If you are a plugin developer, upgrading your module, make sure that you do not have foundation-sites related code.

Reviewer: ```suggestion
If you are a developer or implementer, and you are upgrading your module or application, make sure that you do not have `foundation-sites` related code.
```
